### DISCUSSION TITLE
PlatformDispatcher into its own crate?

### DISCUSSION THREAD

**Discussion by nickpdemarco** [zed-industries/zed#22089] [Feature Requests]
Hello all,

I've been tasked with writing a task scheduler in Rust that dispatches to the platform schedulers on Linux, macOS, and Windows. I was happy to see the Zed team [beat me to it](https://github.com/zed-industries/zed/blob/188c55c8a6673789bd863741d47608f9bb02684e/crates/gpui/src/platform.rs#L448). My team would like to leverage this code, but we're not sure about relying on all of GPUI to do it. Would this group be supportive of a PR(s) to break this logic into its own crate? Or should we look into maintaining our own fork?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
